tests/format/js/import-attributes/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -265,6 +505,69 @@ exports[`quoted-keys.js [meriyah] format 1`] = `
 Cause: [1:32]: Unexpected token: 'with'"
 `;
 
+exports[`quoted-keys.js - {"quoteProps":"consistent"} [acorn] format 1`] = `
+"Unexpected token (1:29)
+> 1 | import json from './mod.js' with { "type":"json" };
+    |                             ^
+  2 | import json from './mod.js' with { "foobar":"json" };
+  3 | import json from './mod.js' with { "type-foo" :"json" };
+  4 | import json from './mod.js' with { "333foo" :"json" };
+Cause: Unexpected token (1:28)"
+`;
+
+exports[`quoted-keys.js - {"quoteProps":"consistent"} [espree] format 1`] = `
+"Unexpected token with (1:29)
+> 1 | import json from './mod.js' with { "type":"json" };
+    |                             ^
+  2 | import json from './mod.js' with { "foobar":"json" };
+  3 | import json from './mod.js' with { "type-foo" :"json" };
+  4 | import json from './mod.js' with { "333foo" :"json" };
+Cause: Unexpected token with"
+`;
+
+exports[`quoted-keys.js - {"quoteProps":"consistent"} [meriyah] format 1`] = `
+"Unexpected token: 'with' (1:32)
+> 1 | import json from './mod.js' with { "type":"json" };
+    |                                ^
+  2 | import json from './mod.js' with { "foobar":"json" };
+  3 | import json from './mod.js' with { "type-foo" :"json" };
+  4 | import json from './mod.js' with { "333foo" :"json" };
+Cause: [1:32]: Unexpected token: 'with'"
+`;
+
+exports[`quoted-keys.js - {"quoteProps":"consistent"} format 1`] = `
+====================================options=====================================
+parsers: ["babel", "typescript"]
+printWidth: 80
+quoteProps: "consistent"
+                                                                                | printWidth
+=====================================input======================================
+import json from './mod.js' with { "type":"json" };
+import json from './mod.js' with { "foobar":"json" };
+import json from './mod.js' with { "type-foo" :"json" };
+import json from './mod.js' with { "333foo" :"json" };
+import json from './mod.js' with { "0" :"json" };
+import json from './mod.js' with { "0n" :"json" };
+import json from './mod.js' with { "33333" :"json" };
+import json from './mod.js' with { "33333n" :"json" };
+import json from './mod.js' with { "33333b" :"json" };
+import json from './mod.js' with { "03333" :"json" };
+
+=====================================output=====================================
+import json from "./mod.js" with { type: "json" };
+import json from "./mod.js" with { foobar: "json" };
+import json from "./mod.js" with { "type-foo": "json" };
+import json from "./mod.js" with { "333foo": "json" };
+import json from "./mod.js" with { "0": "json" };
+import json from "./mod.js" with { "0n": "json" };
+import json from "./mod.js" with { "33333": "json" };
+import json from "./mod.js" with { "33333n": "json" };
+import json from "./mod.js" with { "33333b": "json" };
+import json from "./mod.js" with { "03333": "json" };
```

Comment:
I think we need more tests for different `--quote-props` options.

Reply:
[b78e90f](https://github.com/prettier/prettier/pull/15888/commits/b78e90f25226056de6c869411e7aabaac402f940)